### PR TITLE
feat(extras): add more keymaps to util.rest

### PR DESCRIPTION
**Pull request by cbosvik**
## Description

Add keymaps util.rest extra

## Related Issue(s)

n/a

## Checklist

- [x] I've read the [CONTRIBUTING](https://github.com/LazyVim/LazyVim/blob/main/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: lua/lazyvim/plugins/extras/util/rest.lua [hunk 1]
```diff
@@ -9,10 +9,17 @@ return {
     ft = "http",
     keys = {
       { "<leader>R", "", desc = "+Rest" },
+      { "<leader>Rb", "<cmd>lua require('kulala').scratchpad()<cr>", desc = "Open scratchpad" },
+      { "<leader>Rc", "<cmd>lua require('kulala').copy()<cr>", desc = "Copy as cURL" },
+      { "<leader>Ri", "<cmd>lua require('kulala').inspect()<cr>", desc = "Inspect current request" },
+      { "<leader>Rg", "<cmd>lua require('kulala').download_graphql_schema()<cr>", desc = "Download GraphQL schema" },
+      { "<leader>Rr", "<cmd>lua require('kulala').replay()<cr>", desc = "Replay the last request" },
       { "<leader>Rs", "<cmd>lua require('kulala').run()<cr>", desc = "Send the request" },
+      { "<leader>RS", "<cmd>require('kulala').show_stats()<cr>", desc = "Show stats" },
```
Comment (janbua) [inline lua/lazyvim/plugins/extras/util/rest.lua:18]: Did you mean this?

```suggestion
      { "<leader>RS", "<cmd>lua require('kulala').show_stats()<cr>", desc = "Show stats" },
```

Comment (cbosvik) [reply lua/lazyvim/plugins/extras/util/rest.lua]: thanks

**Comment by kevintraver** [comment]
Copy to and from _curl_ could be useful as well:
```
{ "<leader>Ry", "<cmd>lua require('kulala').copy()<cr>", desc = "Copy curl command" },
{ "<leader>Rp", "<cmd>lua require('kulala').from_curl()<cr>", desc = "Paste from curl" },
```

**Comment by cbosvik** [comment]
> Paste from _curl_ could be useful as well:
> 
> ```
> { "<leader>Rp", "<cmd>lua require('kulala').from_curl()<cr>", desc = "Paste from curl" },
> ```

added that too, thanks

**Comment by folke** [comment]
please resolve the conflicts

**Comment by folke** [comment]
ty

**Pull request merged**